generator/internal/metadata_decorator_generator.cc

```diff
@@ -365,6 +368,9 @@ void $metadata_class_name$::SetMetadata(grpc::ClientContext& context,
 
 void $metadata_class_name$::SetMetadata(grpc::ClientContext& context) {
   context.AddMetadata("x-goog-api-client", api_client_header_);
+  for (auto const& kv : fixed_metadata_) {
+    context.AddMetadata(kv.first, kv.second);
+  }
```

Comment:
Do we have any idea about which value (first or last) would prevail in the service should the same key be repeated?

Given such knowledge, and the knowledge that iteration order of a `multimap` is insertion order for equal keys, we should be deliberate in the order of  `AddMetadata()` calls between `fixed_metadata_` and the two literal keys.

Reply:
> we should be deliberate

I think we can be blasé about this. As far as we know, the only fixed keys are "bigtable_features" and "google-cloud-resource-prefix". I don't think we will accidentally cause conflicts with the "x-goog-*" keys.

Reply:
OK.  I think the new ordering is at least more locally consistent.  Thanks.